addons/project/static/src/scss/project_rightpanel.scss

```diff
@@ -162,7 +162,7 @@ html .o_web_client > .o_action_manager > .o_action > .o_content > .o_controller_
                     border-width: 1px;
                     margin-top: -1px;
                     margin-left: -1px;
-                    height: 56px;
+                    height: 57.5px;
```

Comment:
? 🤔

Reply:
To align kanban list card to stat button of right side panel [Image](https://aju.nimbusweb.me/s/share/6473176/x98i9bhmvvy1gei3ylb7)